src/libecalc/core/graph_result.py

```diff
@@ -215,50 +216,90 @@ def _parse_emissions(emissions: Dict[str, EmissionResult]) -> Dict[str, libecalc
             for key in emissions
         }
 
+    @staticmethod
+    def get_operational_setting_index(timestep: datetime, operational_settings_used: TimeSeriesInt):
+        timestep_index = operational_settings_used.timesteps.index(timestep)
+
+        operational_setting_index = operational_settings_used.values[timestep_index]
```

Comment:
I would prefer to use a different name. This is easily confused with the index in the list. How about `operational_setting_used`? Or perhaps you have sth better?

Reply:
or Id?

Reply:
Perfect

Reply:
I like `operational_setting_used`, as it refers to one entry in the in the time series `operational_settings_used`, or?